mne/dipole.py

```diff
@@ -386,11 +400,18 @@ def constraint(rd):
     amp = np.sqrt(np.sum(Q * Q))
     norm = 1 if amp == 0 else amp
     ori = Q / norm
+
+    dist = _compute_nearest(surf['rr'], rd_final[np.newaxis, :],
+                            return_dists=True)[1][0]
+    logger.info('---- Fitted : %7.1f ms, distance to inner skull : %2.4f mm' %
+                (1000 * t, dist * 1000))
     return rd_final, amp, ori, gof, residual
 
 
 @verbose
-def fit_dipole(evoked, cov, bem, trans=None, n_jobs=1, verbose=None):
+def fit_dipole(evoked, cov, bem, trans=None, min_dist=5,
```

Comment:
Not with any of our standard BEMs at least 5mm for our standard BEM. Ideally someone (not volunteering) would write a simple feature that calculates the average distance between neighboring vertices in the BEM and uses that to suggest the correct default. using the suggestions from Alex's OPENMEEG paper which I believe said for our type of BEM the BEM intervertex distance should be roughly equivalent to the mindist